### PR TITLE
Improve readability of WASM testsuite definition

### PR DESCRIPTION
**Pull request by electrum**
Changes:

- Use sorted XML list in POM for cleaner diffs
- Rename parameters for clarity on wast file vs test
- Enforce that all wast files are explicitly included or excluded
 
This will ensure that we aren't missing any tests by accident.

### CODE REVIEW

**Comment by electrum** [comment]
If you want to merge yours first, this is easy for me to rebase. Up to you. This change should make future conflicts easier to deal with.

**Comment by andreaTP** [comment]
> If you want to merge yours first, this is easy for me to rebase.

Thanks, if it doesn't take too long to have the approvals I'll try to go down this path.

**Comment by andreaTP** [comment]
@electrum please rebase and we move on with this one too, sorry for asking you to resolve the conflicts this time and thanks for your patience.

**Comment by electrum** [comment]
@andreaTP no problem, I rebased it

**Review by andreaTP — APPROVED** [review]
LGTM, thanks! Keep those improvements coming! :-)

File: runtime/pom.xml [hunk 1]
```diff
@@ -41,152 +41,226 @@
...
-            table-sub.wast,
-            table_fill.wast,
-            table_get.wast,
-            table_grow.wast,
-            table_init.wast,
-            table_set.wast,
-            table_size.wast,
-            unreached-invalid.wast</excludedInvalidTests>
+          <includedWasts>
+            <wast>address.wast</wast>
```
Comment (bhelx) [inline runtime/pom.xml:45]: Thank you, this should make diffs and reading this a lot cleaner!

**Pull request merged**